index.src.html

```diff
@@ -298,10 +298,10 @@ <h3 id="queue-report" algorithm>
     Queue |data| as |type| for |destination|
   </h3>
 
-  Given a serializable object (|data|), a string (|type|), another string
+  To <dfn export lt="queue" for="reporting">queue</dfn> a <a>report</a> given a
```

Comment:
`lt="queue"` isn't necessary when it exactly matches the contained text, right?

Reply:
Correct, sorry about that.